modules/outbreak_alerts/mmd.json

```diff
@@ -848,7 +1010,7 @@
             "skipSerialization": false,
             "displayName": "{{ latestAnnouncement }}",
```

Comment:
What about this name?

Reply:
we mapped fortiguard field name with FSR fiels